vtds_cluster_kvm/private/scripts/node_builder.py

```diff
@@ -0,0 +1,370 @@
+#! python
```

Comment:
REVIEWERS: this file defines the Node Builder base class and the two new Node Builder class implementations added by this PR. It is one of the python libraries sent to the blade during the `prepare` phase.